src/get-started/create-service.md

```diff
@@ -10,6 +10,8 @@ creates a wallet with a default balance and transfers money between wallets.
 
 You can view and download the full source code of this tutorial
 [here][cryptocurrency].
+There the code is divided into separate modules
```

Comment:
IMO this sentence is too vague and brings more questions than information: what do you mean by a module, which areas of responsibilities, etc.? I'd remove it.